Tests/TuistCoreIntegrationTests/RootDirectoryLocatorTests.swift

```diff
@@ -168,4 +168,30 @@ final class RootDirectoryLocatorTests: TuistTestCase {
             "APlugin/",
         ].map { temporaryDirectory.appending(try RelativePath(validating: $0)) })
     }
+
+    func test_locate_when_a_spm_manifest_file_exists() async throws {
```

Comment:
nit: it was recently pointed out to me that the SPM abbreviation should not be used, so let's move away from it: https://elk.zone/mastodon.online/@maxd@mastodon.social/113633867048232923
```suggestion
    func test_locate_when_a_swiftpm_manifest_file_exists() async throws {
```

Reply:
got it!